etcdctl/ctlv3/command/ep_command.go

```diff
@@ -206,13 +217,20 @@ func epHashKVCommandFunc(cmd *cobra.Command, args []string) {
 	}
 }
 
-func endpointsFromCluster(cmd *cobra.Command) []string {
+type memberInfo struct {
+	ID         uint64
+	ClientURLs []string
+}
+
+func endpointsFromCluster(cmd *cobra.Command) []memberInfo {
+	var memberInfos []memberInfo
 	if !epClusterEndpoints {
 		endpoints, err := cmd.Flags().GetStringSlice("endpoints")
```

Comment:
refer https://github.com/coreos/etcd/pull/9780
you can trim the `endpoints` flag value.